contracts/src/RolldownStorage.sol

```diff
@@ -22,6 +22,12 @@ abstract contract RolldownStorage is IRolldown {
     mapping(uint256 => L2UpdatesToRemove) internal l2UpdatesToRemove;
     mapping(address => uint) public pendingEthWithdrawals;   
 
+    // TODO: check how to align __gap array size
```

Comment:
@iStrike7 any idea how `gap` array should be removed ?

Reply:
@mateuszaaa after one version of the contract is deployed and you want to upgrade it to the next version with more storage items, you need to add the storage items at the end of the storage item list (to not change the storage layout - which depends on sequence), and when you add storage items at the end you need to remove as many from _gap... This _gap is used to preserve the storage layout of the contract that inherits from the contract you are modifying...

Reply:
But you don't really need to do this now since, we don't have this deployed anywhere (I don't testnet counts here since that will be fully refreshed with v3), so you can just leave _gap as is and add/change the storage layout as you like...